cmd/snap-update-ns/bootstrap.c

```diff
@@ -351,7 +352,7 @@ static int parse_arg_u(int argc, char * const *argv, int *optind, unsigned long
 			|| (*uid_text != '\0' && uid_text_end != NULL
 				&& *uid_text_end != '\0')) {
 		bootstrap_msg = "cannot parse user id";
-		bootstrap_errno = errno;
```

Comment:
Sorry, I'm not sure I understand this - AFAICT nothing in the `if( (parsed_uid ...)` changes errno, so why do we need to safe it?

Reply:
We don't need to save it yet, as I said above this is in case someone forgets about this fact and adds any code in between, like I did while debugging isspace.

Reply:
You mean if someone adds debug code into the `if()` ? Sorry for being so slow today, it seems the save/restore is so close together that I don't quite see a use case. Maybe an example would help me :) ?

Reply:
Yes, exactly for that kind of case. It's not mandatory, we can drop this patch if you feel like it.